terraform/20-app/main.tf

```diff
@@ -28,3 +35,53 @@ terraform {
     key            = "app/state.tfstate"
   }
 }
+
+module "cognito" {
+  source = "../modules/cognito"
+
+  user_pool_name    = "app-${terraform.workspace}-user-pool"
+  client_name       = "app-${terraform.workspace}-client"
+  user_pool_domain  = "app-${terraform.workspace}-auth"
+
+  callback_urls = ["https://${terraform.workspace}.dev.ukhsa-dashboard.data.gov.uk/callback"]
+  logout_urls   = ["https://${terraform.workspace}.dev.ukhsa-dashboard.data.gov.uk/logout"]
+}
+
+resource "aws_security_group" "app_security_group" {
```

Comment:
We've also tended to lean on pre-made modules where we can.
So in this instance we'd use `"terraform-aws-modules/security-group/aws"` module

Reply:
Nice, amended to what I think it should be